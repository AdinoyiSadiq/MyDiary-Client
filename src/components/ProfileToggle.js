import React from 'react';

const ProfileToggle = () => (
  <div className="sectionToggle">
		<a className="active" id="profileToggle">My Profile</a>
		<a id="reminderToggle">Add Reminder</a>
	</div>
);

export default ProfileToggle;