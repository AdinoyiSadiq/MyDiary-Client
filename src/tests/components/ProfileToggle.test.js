import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ProfileToggle from '../../components/ProfileToggle';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<ProfileToggle />);
});


describe('ProfileToggle UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});