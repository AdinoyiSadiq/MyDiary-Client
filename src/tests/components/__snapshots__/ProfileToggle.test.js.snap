// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileToggle UI render features view should render as expected 1`] = `
<div
  className="sectionToggle"
>
  <a
    className="active"
    id="profileToggle"
  >
    My Profile
  </a>
  <a
    id="reminderToggle"
  >
    Add Reminder
  </a>
</div>
`;
